src/main/java/com/github/bartimaeusnek/cropspp/crops/TC/BasicManaBeanCrop.java

```diff
@@ -49,41 +65,34 @@ public int stat(int n) {
         }
     }
 
-    @Override
-    public boolean canGrow(ICropTile crop) {
-        if (blockCrystal == null) blockCrystal = BlockGetterTC.getBlock_asBlock("blockCrystal", 0);
-        if (crop.getSize() == maxSize()) return false;
-        if (crop.getSize() > 1) {
-            return crop.isBlockBelow(blockCrystal);
-        } else return true;
-    }
-
     @Override
     public int weightInfluences(ICropTile crop, float humidity, float nutrients, float air) {
         // Requires no humidity but nutrients.
-        return (int) ((double) humidity * 1.3 + (double) nutrients * 1 + (double) air * 0.7);
+        // Doc seems incorrect, but anyhow i've inverted it for safety sake
+        return (int) ((double) humidity / 1.3D + (double) nutrients + (double) air * 0.7);
```

Comment:
Also make air requirement division maybe?

Reply:
addressed in https://github.com/GTNewHorizons/Crops-plus-plus/pull/57/commits/363580f75b54ddb4688ce03d45a3a68421d83faa